### ISSUE TITLE
[Bug] Crash app during the opening. Unity/ ExecutorUtils$1$1.onRun

### ISSUE THREAD

**Issue by Maskusa** [firebase/firebase-unity-sdk#567]
### [REQUIRED] Please fill in the following fields:

  * Unity editor version: 2021.3.12f1
  * Firebase Unity SDK version: 10.2.0 
  * Source you installed the SDK: .unitypackage
  * Problematic Firebase Component: FirebaseCrashlytics
  * Other Firebase Components in use: FirebaseAnalytics, FirebaseCrashlytics, FirebaseDatabase
  * Additional SDKs you are using: AdMob, YandexMobile
  * Platform you are using the Unity editor on: Windows
  * Platform you are targeting: Android
  * Scripting Runtime: Mono, and IL2CPP
  * Pre-built SDK from the [website](https://firebase.google.com/download/unity) or open-source from this repo: This repo

### [REQUIRED] Please describe the issue here:

I'm getting a Crashlytics report about an app crashing while opening. I don't have more information.
How can this be fixed?

**firebase_unity_sdk_10.2.0**
**Full Stack trace:** https://drive.google.com/file/d/1NGlxJiyyFID14iNkD3YLFtnp0oPsPChU/view?usp=sharing
```
ExecutorUtils.java line 64
com.google.firebase.crashlytics.internal.common.ExecutorUtils$1$1.onRun
```

```
Fatal Exception: java.lang.Error: FATAL EXCEPTION [UnityMain]
Unity version     : 2021.3.12f1
Device model      : unknown GCE x86 phone
Device fingerprint: generic/gce_x86_phone/gce_x86:9/PGR1.190916.001/5877764:userdebug/test-keys
Build Type        : Release
Scripting Backend : IL2CPP
ABI               : armeabi-v7a
Strip Engine Code : true
```

```
Caused by java.lang.Error: *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
Version '2021.3.12f1 (8af3c3e441b1)', Build type 'Release', Scripting Backend 'il2cpp', CPU 'armeabi-v7a'
Build fingerprint: 'generic/gce_x86_phone/gce_x86:9/PGR1.190916.001/5877764:userdebug/test-keys'
Revision: '0'
ABI: 'arm'
Timestamp: 2022-12-12 07:01:40-0800
pid: 8451, tid: 9151, name: UnityMain  >>> com.subtit.player <<<
uid: 10093
signal 11 (SIGSEGV), code 2 (SEGV_ACCERR), fault addr 0x0
    r0  00000001  r1  b900ad20  r2  00003435  r3  00000000
    r4  00000000  r5  00000001  r6  00000001  r7  b7797420
    r8  00000002  r9  b779c37c  r10 00000000  r11 b7796eec
    ip  e969a2d0  sp  bae7f3e0  lr  c0b7d88c  pc  00000000


```

```
awaitEvenIfOnMainThread task continuation executor1:
       at java.lang.Object.wait(Object.java)
       at java.lang.Thread.parkFor$(Thread.java:2137)
       at sun.misc.Unsafe.park(Unsafe.java:358)
       at java.util.concurrent.locks.LockSupport.park(LockSupport.java:190)
       at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2059)
       at java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:442)
       at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1092)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
       at com.google.firebase.crashlytics.internal.common.ExecutorUtils$1$1.onRun(ExecutorUtils.java:64)
       at com.google.firebase.crashlytics.internal.common.BackgroundPriorityRunnable.run(BackgroundPriorityRunnable.java:27)
       at java.lang.Thread.run(Thread.java:764)
```

**Comment by oriontvv** [comment]
I got the same for `unity 2021.3.14f1` and `crashlytics 9.1.0`

**Comment by paulinon** [comment]
Hi @Maskusa,

In order to identify what's causing this, could you try adding the following line of code then share the logs you received during the crash:

`Firebase.FirebaseApp.LogLevel = Firebase.LogLevel.Debug;`

**Comment by Maskusa** [comment]
I had another slightly similar bug that crashes on startup. 
[https://github.com/firebase/firebase-unity-sdk/issues/561](url)
This is Yandex SDK. I noticed that the problems have a common symptom - Target platform "x86". I got a response from their tech support that I need to check the configuration of the Unity editor. Flags should be active there (Chrome OS). I've always compiled AAB builds on the target workstation platforms, flags (Chrome OS) were disabled. Perhaps this is the key to solving the problem. Right now I'm testing AAB builds with all flags active. ARM+ (Chrome OS)
